app/services/claims/fee_calculator/calculate_price.rb

```diff
@@ -50,8 +59,13 @@ def setup(options)
         @advocate_category = options[:advocate_category] || claim.advocate_category
         @quantity = options[:quantity] || 1
         @current_page_fees = options[:fees].values
+        exclusions
       rescue StandardError
-        raise 'incomplete'
+        raise Exceptions::InsufficientData
+      end
+
+      def exclusions
+        raise 'implement in subclass'
```

Comment:
This is untested. Part of the -0.4 coverage decrease?

Reply:
@colinbruce I have improved test coverage on this, but did not make any noticable difference to total coverage, so  have been doing local test coverage vetting and improving generally in my spare moments. will push those over the weekend probably.

Reply:
Cool. That seems to be the only line that Code Climate is concerned about... it's a 0.4 reduction.  I'm happy to ignore if you are!

Reply:
@colinbruce Morning, so i have unskipped a couple of `unit_price_spec` tests that were causing problems previously (flickering coz of FeeTypeMappings Singleton class i discovered). Plus to handle codeclimate coverage I have added a couple of commits to generally improve coverage and config for simplecov.

If you want to take a look at those, please do. Otherwise am going to merge and deploy